environment.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+import time
+
+from color_refinement import get_number_automorphisms, is_isomorphisms
+from graph_io import load_graph
+
+PATH_BRANCHING = 'graphs/branching'
+PATH_COLORREF = 'graphs/colorref'
+PATH_TREEPATHS = 'graphs/treepaths'
+OUTPUT_PATH = 'export/result'
+TIME_ROUND = 5
+
+FILE = 'graphs/colorref/colorref_smallexample_4_16.grl'
+
+
+def main():
+    if FILE == '':
+        graph_files = find_graphs()
+    else:
+        graph_files = [FILE]
+    process_graph_files(graph_files)
+
+
+def find_graphs():
+    branching_graphs = [PATH_BRANCHING + '/' + file for file in os.listdir(PATH_BRANCHING)]
```

Comment:
Nb: er zitten ook niet graph files in die mapjes. Misschien nog een check of het op 'gr' of 'grl' eindigt?